devops: re-factor list-dependencies script to output per-browser results

aslushnikov opened the pull request:
This patch:
- refactors script to output per-browser package dependencies. This is to aid with
  per-browser docker setup and Playwright github action.
- sorts package maps for both Ubuntu 18.04 and Ubuntu 20.04 alphabetically (and removes a stray dependency)

References #2926

yury-s commented on utils/linux-browser-dependencies/ubuntu-18-04-run-result:
```diff
@@ -0,0 +1,319 @@
+Working on: Ubuntu 18.04.4 LTS
```
Why do we need these logs in the repo?

aslushnikov replied:
removed them.

aslushnikov committed:
devops: re-factor list-dependencies script to output per-browser results

This patch:
- refactors script to output per-browser package dependencies. This is to aid with
  per-browser docker setup and Playwright github action.
- sorts package maps for both Ubuntu 18.04 and Ubuntu 20.04 alphabetically (and removes a stray dependency)
- hardcodes `DLOPEN` libraries for the browsers

References #2926
